orttraining/orttraining/eager/test/ort_ops.py

```diff
@@ -4,11 +4,49 @@
 # pylint: disable=missing-docstring
 
 import unittest
+from parameterized import parameterized
 
 import numpy as np
 import onnxruntime_pybind11_state as torch_ort
 import torch
 
+ops= [["abs",         torch.tensor([-1, -2, 3, -6, -7])],
```

Comment:
Would it be useful to include a comment describing what the structure of `ops` is? (e.g. what are the values of the tuple?)